core/static/js/dit.class.modal.js

```diff
@@ -154,7 +151,7 @@
       } else if (e.which === 9 && !e.shiftKey) {
       // After pressing tab when close button is selected
       // loop back around to the first link
-        self.$firstLink.focus();
+        self.focus();
```

Comment:
Since you're adding the loop back code directly to self.$closeButton, doesn't that mean there's no point in finding and declaring a self.$lastFocusElement?

I might be missing it, but it doesn't look like $lastFocusElement is being used anywhere.